src/index.ts

```diff
@@ -137,9 +163,17 @@ const addOAuthInterceptor = (
       oauthUrl.toString(),
       paramsToSign,
       secret,
-      token
+      tokenSecret || token
```

Comment:
Looking at the docs I wonder if there's ever actually a case where `token` should be used here.  I probably just did this incorrectly, and should always have passed a token secret separately from the token to use for the signature calculation.

```suggestion
      tokenSecret
```

Reply:
I originally left `token` in as a fallback to not break existing calls, but I do agree that it should always have been the token secret.